src/namespaceDetector.ts

```diff
@@ -33,9 +33,9 @@ export default class NamespaceDetector {
 
         const csprojFile = csprojs[0];
         const fileContent = await this.read(Uri.file(csprojFile));
-        const rootNamespace = new CsprojReader(fileContent).getRootNamespace();
+        let rootNamespace = new CsprojReader(fileContent).getRootNamespace();
         if (rootNamespace === undefined) {
-            return undefined;
+            rootNamespace = path.basename(csprojFile, '.csproj')
```

Comment:
This edit is familiar but why is it in this PR?

Reply:
you're right.
While debugging, I noticed that I got some weird namespaces and that fixed it.
I reverted.

Reply:
Ah, okey. For some reason the `let rootNamespace` didn't reverted back to `const rootNamespace`. Can You fix it, please?